roles/neutron-common/templates/etc/neutron/policy.json

```diff
@@ -36,8 +46,11 @@
     "get_network:provider:physical_network": "rule:admin_only",
     "get_network:provider:segmentation_id": "rule:admin_only",
     "get_network:queue_id": "rule:admin_only",
+    "get_network_ip_availabilities": "rule:admin_only",
+    "get_network_ip_availability": "rule:admin_only",
```

Comment:
We are considering making this available in 2Q release. It should be open to cloud_admin as well I would think.